### PR TITLE
[PUX-2580] | Refactor iOS code

### PR DESCRIPTION
**Pull request by TheInkedEngineer**

### CODE REVIEW

File: ios/WebViewExample.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -466,6 +443,7 @@
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
+				DEVELOPMENT_TEAM = 6X6F37SBAW;
```
Comment (tl-daven-gomes) [inline ios/WebViewExample.xcodeproj/project.pbxproj:446]: dev team was left in

File: ios/WebViewExample.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -439,6 +415,7 @@
...
```
Comment (tl-daven-gomes) [inline ios/WebViewExample.xcodeproj/project.pbxproj:418]: dev team was left in

File: ios/WebViewExample.xcodeproj/xcuserdata/firas.safa.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tl-daven-gomes) [inline ios/WebViewExample.xcodeproj/xcuserdata/firas.safa.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist:1]: probably worth removing breakpoints?

**Review by tl-daven-gomes — APPROVED** [review]

File: ios/WebViewExample/WebViewController.swift [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+        } else {
+            decisionHandler(.allow)
+        }
+    }
+}
+
+extension WebView: WKUIDelegate {
+    func webView(_ webView: WKWebView, createWebViewWith configuration: WKWebViewConfiguration, for navigationAction: WKNavigationAction, windowFeatures: WKWindowFeatures) -> WKWebView? {
+        if let url = navigationAction.request.url, url.scheme != "http" && url.scheme != "https" && url.scheme != "about" {
+            UIApplication.shared.open(url, options: [:], completionHandler: nil)
```
Comment (tl-sebastien-kovacs) [inline ios/WebViewExample/WebViewController.swift:110]: Did we managed to understand why this is needed ?
I suspect this is the part that opens the bank link outside the webview, correct ?

**Review by tl-sebastien-kovacs — DISMISSED** [review]
Except my minor comment looks good ✔️

**Comment by tl-sebastien-kovacs** [comment]
Actually could we also update the Readme with updated screenshots ?

File: ios/WebViewExample/WebViewController.swift (hunk 4 above)
Comment (TheInkedEngineer) [reply ios/WebViewExample/WebViewController.swift:110]: yeah without it we can't basically navigate inside the WebView. And this bit is to make sure that a certain scheme is fallowed as a safety net.

**Pull request merged**